src/main/java/com/incognia/transaction/RegisterPaymentRequest.java

```diff
@@ -1,5 +1,8 @@
-package com.incognia;
+package com.incognia.transaction;
```

Comment:
Shouldn't this class be on transaction.payment?

Reply:
you're right

Reply:
done